05_iigo/index.tex

```diff
@@ -347,52 +348,69 @@ \section{Accountability Cycle}
+
+\begin{rule_IIGO} \label{rule:roles_must_hold_election}
+    The role $X$ has the \emph{obligation} to conduct a vote for the election of $Y$ if and only if $Y$ has been in power for more turns than the turn length or if role $Z$ has made a monitoring announcement that indicates wrongdoing by $Y$
+\end{rule_IIGO}
+
+\begin{rule_IIGO} \label{rule:must_appoint_elected_island}
+    The role $X$ has the \emph{permission} to \emph{declare the winner} $W$ for an election $E$ if $W = W^{*}$, where $W^{*}$ is the winner produced by \emph{calling} a vote for the election $E$.
+\end {rule_IIGO}
+
+Unlike the speaker, the process of election is more regimented. The power of calling a vote for an election and the power of declaring the result are combined into one action. The decision came as a result of motion to simplify the system for implementation. However, the powers are still, somewhat, kept separate. When facilitating an election the roles still have the option to declare a winner of their choosing. Rule~\ref{rule:must_appoint_elected_island} is what governs this choice. The agents don't, however, have the option to not declare a result at all, holding an election will always result in an declaration of the winner.
 
 \section{Budget and Salary}
 \subsection{Budget}
 %Actions associated with the IIGO have an associated cost that is defined as a configuration parameter. The institutional-power-enabled actions of  identified to require a "computational" component are:
 
-Institutional-enabled-power actions in the IIGO have an associated cost. Some examples of such actions include:
+For the simulation we have defined the resources gathered by the islands to be endogenous. Hence we assume that self-organization will consume those resources and institutional-power-enabled actions in the IIGO have an associated cost. The institutional-power actions with such a cost are:
 
 
 %that is defined as a configuration parameter. The institutional-power-enabled actions of  identified to require a "computational" component are:
 
 
-%We have defined the resource to be an endogenous one, hence any computation surrounding the distribution of the resource must use up some of that resource. 
+%We have defined the resource to be an endogenous one, hence any computation surrounding the distribution of the resource must use up some of that resource.
 \begin{itemize}
-\item Calling any vote and computing the winner.
+\item President selecting a rule from the rule proposal list.
+\item President deciding the amount of individual taxation.
```

Comment:
```suggestion
\item President deciding the amount of individual taxation for each island.
```

Reply:
individual implies for each islands

Reply:
well..... this is clearer I would say

Reply:
what is 'individual' in our CW is a bit vague

Reply:
Don't want to use for each island because the way we have it in code, you either decide the tax for all or for none. For each island makes it sound like that action is individual rather than the tax.
```suggestion
\item President deciding the amount of taxation.
```